<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ClientAttachSyncDialog</class>
 <widget class="QDialog" name="ClientAttachSyncDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>560</width>
    <height>364</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="0" colspan="2">
    <widget class="CProgressBar" name="progressBar">
     <property name="value">
      <number>24</number>
     </property>
    </widget>
   </item>
   <item row="1" column="1">
    <widget class="QGroupBox" name="grpDeattached">
     <property name="title">
      <string>Откреплённые</string>
     </property>
     <layout class="QGridLayout" name="gridLayout_5">
      <item row="0" column="0">
       <layout class="QGridLayout" name="gridLayout_4">
        <item row="3" column="0">
         <widget class="QLabel" name="lblAlreadyDeattachedD">
          <property name="text">
           <string>Уже откреплено ранее</string>
          </property>
          <property name="buddy">
           <cstring>edtAlreadyDeattachedD</cstring>
          </property>
         </widget>
        </item>
        <item row="5" column="0">
         <widget class="QLabel" name="lblNoAttachD">
          <property name="text">
           <string>Прикрепление не найдено</string>
          </property>
          <property name="buddy">
           <cstring>edtNoAttachD</cstring>
          </property>
         </widget>
        </item>
        <item row="4" column="0">
         <widget class="QLabel" name="lblNotDeattachedD">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="text">
           <string>Не откреплено</string>
          </property>
          <property name="buddy">
           <cstring>edtNotDeattachedD</cstring>
          </property>
         </widget>
        </item>
        <item row="0" column="1">
         <widget class="QLineEdit" name="edtSelectedD">
          <property name="font">
           <font>
            <weight>75</weight>
            <bold>true</bold>
           </font>
          </property>
          <property name="text">
           <string>0</string>
          </property>
          <property name="alignment">
           <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
          </property>
          <property name="readOnly">
           <bool>true</bool>
          </property>
         </widget>
        </item>
        <item row="0" column="0">
         <widget class="QLabel" name="lblSelectedD">
          <property name="text">
           <string>Выбрано</string>
          </property>
          <property name="buddy">
           <cstring>edtSelectedD</cstring>
          </property>
         </widget>
        </item>
        <item row="2" column="0">
         <widget class="QLabel" name="lblDeattachedD">
          <property name="text">
           <string>Откреплено</string>
          </property>
          <property name="buddy">
           <cstring>edtDeattachedD</cstring>
          </property>
         </widget>
        </item>
        <item row="1" column="0">
         <widget class="QLabel" name="lblSyncedD">
          <property name="text">
           <string>Синхронизировано</string>
          </property>
          <property name="buddy">
           <cstring>edtSyncedD</cstring>
          </property>
         </widget>
        </item>
        <item row="6" column="0">
         <widget class="QLabel" name="lblNotFoundD">
          <property name="text">
           <string>Пациента нет в БД</string>
          </property>
          <property name="buddy">
           <cstring>edtNotFoundD</cstring>
          </property>
         </widget>
        </item>
        <item row="1" column="1">
         <widget class="QLineEdit" name="edtSyncedD">
          <property name="font">
           <font>
            <weight>75</weight>
            <bold>true</bold>
           </font>
          </property>
          <property name="text">
           <string>0</string>
          </property>
          <property name="alignment">
           <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
          </property>
          <property name="readOnly">
           <bool>true</bool>
          </property>
         </widget>
        </item>
        <item row="2" column="1">
         <widget class="QLineEdit" name="edtDeattachedD">
          <property name="font">
           <font>
            <weight>75</weight>
            <bold>true</bold>
           </font>
          </property>
          <property name="text">
           <string>0</string>
          </property>
          <property name="alignment">
           <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
          </property>
          <property name="readOnly">
           <bool>true</bool>
          </property>
         </widget>
        </item>
        <item row="3" column="1">
         <widget class="QLineEdit" name="edtAlreadyDeattachedD">
          <property name="font">
           <font>
            <weight>75</weight>
            <bold>true</bold>
           </font>
          </property>
          <property name="text">
           <string>0</string>
          </property>
          <property name="alignment">
           <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
          </property>
          <property name="readOnly">
           <bool>true</bool>
          </property>
         </widget>
        </item>
        <item row="4" column="1">
         <widget class="QLineEdit" name="edtNotDeattachedD">
          <property name="font">
           <font>
            <weight>75</weight>
            <bold>true</bold>
           </font>
          </property>
          <property name="text">
           <string>0</string>
          </property>
          <property name="alignment">
           <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
          </property>
          <property name="readOnly">
           <bool>true</bool>
          </property>
         </widget>
        </item>
        <item row="5" column="1">
         <widget class="QLineEdit" name="edtNoAttachD">
          <property name="font">
           <font>
            <weight>75</weight>
            <bold>true</bold>
           </font>
          </property>
          <property name="text">
           <string>0</string>
          </property>
          <property name="alignment">
           <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
          </property>
          <property name="readOnly">
           <bool>true</bool>
          </property>
         </widget>
        </item>
        <item row="6" column="1">
         <widget class="QLineEdit" name="edtNotFoundD">
          <property name="font">
           <font>
            <weight>75</weight>
            <bold>true</bold>
           </font>
          </property>
          <property name="text">
           <string>0</string>
          </property>
          <property name="alignment">
           <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
          </property>
          <property name="readOnly">
           <bool>true</bool>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item row="2" column="0">
       <widget class="QCheckBox" name="chkDeattachOnlyOnDeath">
        <property name="text">
         <string>Откреплять только умерших</string>
        </property>
       </widget>
      </item>
      <item row="1" column="0">
       <spacer name="verticalSpacer_2">
        <property name="orientation">
         <enum>Qt::Vertical</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>20</width>
          <height>40</height>
         </size>
        </property>
       </spacer>
      </item>
     </layout>
     <zorder>chkDeattachOnlyOnDeath</zorder>
     <zorder>verticalSpacer_2</zorder>
    </widget>
   </item>
   <item row="1" column="0">
    <widget class="QGroupBox" name="grpAttached">
     <property name="title">
      <string>Прикреплённые</string>
     </property>
     <layout class="QGridLayout" name="gridLayout_3">
      <item row="1" column="0">
       <spacer name="verticalSpacer">
        <property name="orientation">
         <enum>Qt::Vertical</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>20</width>
          <height>40</height>
         </size>
        </property>
       </spacer>
      </item>
      <item row="2" column="0">
       <widget class="QCheckBox" name="chkCreateClientAttach">
        <property name="text">
         <string>Добавлять прикрепления из ТФОМС</string>
        </property>
       </widget>
      </item>
      <item row="0" column="0">
       <layout class="QGridLayout" name="gridLayout_2">
        <item row="4" column="0">
         <widget class="QLabel" name="lblNotFoundA">
          <property name="text">
           <string>Пациента нет в БД</string>
          </property>
          <property name="buddy">
           <cstring>edtNotFoundA</cstring>
          </property>
         </widget>
        </item>
        <item row="2" column="1">
         <widget class="QLineEdit" name="edtAttachedA">
          <property name="font">
           <font>
            <weight>75</weight>
            <bold>true</bold>
           </font>
          </property>
          <property name="text">
           <string>0</string>
          </property>
          <property name="alignment">
           <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
          </property>
          <property name="readOnly">
           <bool>true</bool>
          </property>
         </widget>
        </item>
        <item row="1" column="0">
         <widget class="QLabel" name="lblSyncedA">
          <property name="text">
           <string>Синхронизировано</string>
          </property>
          <property name="buddy">
           <cstring>edtSyncedA</cstring>
          </property>
         </widget>
        </item>
        <item row="1" column="1">
         <widget class="QLineEdit" name="edtSyncedA">
          <property name="font">
           <font>
            <weight>75</weight>
            <bold>true</bold>
           </font>
          </property>
          <property name="text">
           <string>0</string>
          </property>
          <property name="alignment">
           <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
          </property>
          <property name="readOnly">
           <bool>true</bool>
          </property>
         </widget>
        </item>
        <item row="2" column="0">
         <widget class="QLabel" name="lblAttachedA">
          <property name="text">
           <string>Прикреплено</string>
          </property>
          <property name="buddy">
           <cstring>edtAttachedA</cstring>
          </property>
         </widget>
        </item>
        <item row="0" column="1">
         <widget class="QLineEdit" name="edtSelectedA">
          <property name="font">
           <font>
            <weight>75</weight>
            <bold>true</bold>
           </font>
          </property>
          <property name="text">
           <string>0</string>
          </property>
          <property name="alignment">
           <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
          </property>
          <property name="readOnly">
           <bool>true</bool>
          </property>
         </widget>
        </item>
        <item row="0" column="0">
         <widget class="QLabel" name="lblSelectedA">
          <property name="text">
           <string>Выбрано</string>
          </property>
          <property name="buddy">
           <cstring>edtSelectedA</cstring>
          </property>
         </widget>
        </item>
        <item row="3" column="0">
         <widget class="QLabel" name="lblNoAttachA">
          <property name="text">
           <string>Прикрепление не найдено</string>
          </property>
          <property name="buddy">
           <cstring>edtNoAttachA</cstring>
          </property>
         </widget>
        </item>
        <item row="3" column="1">
         <widget class="QLineEdit" name="edtNoAttachA">
          <property name="font">
           <font>
            <weight>75</weight>
            <bold>true</bold>
           </font>
          </property>
          <property name="text">
           <string>0</string>
          </property>
          <property name="alignment">
           <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
          </property>
          <property name="readOnly">
           <bool>true</bool>
          </property>
         </widget>
        </item>
        <item row="4" column="1">
         <widget class="QLineEdit" name="edtNotFoundA">
          <property name="font">
           <font>
            <weight>75</weight>
            <bold>true</bold>
           </font>
          </property>
          <property name="text">
           <string>0</string>
          </property>
          <property name="alignment">
           <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
          </property>
          <property name="readOnly">
           <bool>true</bool>
          </property>
         </widget>
        </item>
        <item row="5" column="0">
         <widget class="QLabel" name="lblClientsCreatedA">
          <property name="text">
           <string>Пациентов создано</string>
          </property>
          <property name="buddy">
           <cstring>edtClientsCreatedA</cstring>
          </property>
         </widget>
        </item>
        <item row="5" column="1">
         <widget class="QLineEdit" name="edtClientsCreatedA">
          <property name="font">
           <font>
            <weight>75</weight>
            <bold>true</bold>
           </font>
          </property>
          <property name="text">
           <string>0</string>
          </property>
          <property name="alignment">
           <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
          </property>
          <property name="readOnly">
           <bool>true</bool>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item row="3" column="0">
       <widget class="QCheckBox" name="chkCreateClients">
        <property name="text">
         <string>Создавать пациента в БД</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item row="2" column="0" colspan="2">
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QPushButton" name="btnSync">
       <property name="text">
        <string>Синхронизировать</string>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer_2">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="btnClose">
       <property name="text">
        <string>Закрыть</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>CProgressBar</class>
   <extends>QProgressBar</extends>
   <header>library/ProgressBar.h</header>
  </customwidget>
 </customwidgets>
 <tabstops>
  <tabstop>edtSelectedA</tabstop>
  <tabstop>edtSyncedA</tabstop>
  <tabstop>edtAttachedA</tabstop>
  <tabstop>edtNoAttachA</tabstop>
  <tabstop>edtNotFoundA</tabstop>
  <tabstop>edtClientsCreatedA</tabstop>
  <tabstop>chkCreateClientAttach</tabstop>
  <tabstop>chkCreateClients</tabstop>
  <tabstop>edtSelectedD</tabstop>
  <tabstop>edtSyncedD</tabstop>
  <tabstop>edtDeattachedD</tabstop>
  <tabstop>edtAlreadyDeattachedD</tabstop>
  <tabstop>edtNotDeattachedD</tabstop>
  <tabstop>edtNoAttachD</tabstop>
  <tabstop>edtNotFoundD</tabstop>
  <tabstop>chkDeattachOnlyOnDeath</tabstop>
  <tabstop>btnSync</tabstop>
  <tabstop>btnClose</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
